src/app/trainee/revalidation-history/revalidation-history.component.scss

```diff
@@ -1,6 +1,10 @@
 @import "~nhsuk-frontend/packages/core/settings/colours";
 @import "../../../scss/utilities/spacing-helpers";
 
+.trainee-history-wrapper {
+  display: flex;
```

Comment:
could probably create a generic `.d-flex` in `layout.scss` to accompany similar block and grid classes

Reply:
will do. good spot